internal/hud/server/server.go

```diff
@@ -8,21 +8,31 @@ import (
 	"github.com/gorilla/mux"
 	_ "github.com/gorilla/websocket"
 	"github.com/windmilleng/tilt/internal/store"
+	"github.com/windmilleng/wmclient/pkg/analytics"
 )
 
+type analyticsPayload struct {
+	Verb string
```

Comment:
it would usually be more idiomatic to lowercase the fields on serialization `json:"verb"`